### ISSUE TITLE
No more possible to fetch API key 

### ISSUE THREAD

**Issue by Kaujo** [Ging-Buh/cachebox#5]
Latest version obtainable under Google Play. Build 3775. Cache box does not allow anymore to fetch a new API key. Thus I can't anymore use my premium account for cache box. I am not a developer but a very longtime user and fan of cachebox app.
BTW the url www.team-cachebox.de. does not show up any more. Did you stop supporting this nice app?
kind regards
John

**Comment by Kaujo** [comment]
Android 13 on Fairphone 4

**Comment by arbor95** [comment]
The last version is not yet at the playstore.
Use dropbox
 http://www.dropbox.com/sh/5jl7hilhlgmzoze/AAAQBsP8YBYjLh3YEifCc91ia?dl=0 

Or better gdrive

https://drive.google.com/drive/folders/1hKRKMQJ7t-lCNKqK3UbCiQ75oulYRYeD

Where you can find the latest dev version.

**Comment by Kaujo** [comment]
Thanks for the reaf fast reply!I've now installed the 3776 from 9th of january 2024.Unfortunately my problem subsists. I can no more access Geocaching.com.For instance, when I try to actualize the currently selected cache, I get an error: "request failed: 401 unauthorized". And I can no more ask for a new API key in the part of the settings menu, where I used to ask for one in such situations.I am a former It teacher and programmer, but I don't want to participate in this cool project as it would require yet another language or environment to be learned. It would be the 18th language in over 44 years of professional life, so I hope you forgive my lazyness in this matterKind regards John Am 25.02.2024 00:52 schrieb Franz Frese ***@***.***>:
The last version is not yet at the playstore.
Use dropbox
http://www.dropbox.com/sh/5jl7hilhlgmzoze/AAAQBsP8YBYjLh3YEifCc91ia?dl=0
Or better gdrive
https://drive.google.com/drive/folders/1hKRKMQJ7t-lCNKqK3UbCiQ75oulYRYeD
Where you can find the latest dev version.

**Comment by arbor95** [comment]
The version is correct. I succeed by
first enter settings, for login, create api key and follow the instructions.

**Comment by tower27** [comment]
I have exactly the same problems since some weeks. I cannot access geocaching.com and I can’t create a API Key.

 

Von: Kaujo ***@***.***> 
Gesendet: Sonntag, 25. Februar 2024 12:19
An: Ging-Buh/cachebox ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [Ging-Buh/cachebox] No more possible to fetch API key (Issue #5)

 

Thanks for the reaf fast reply!I've now installed the 3776 from 9th of january 2024.Unfortunately my problem subsists. I can no more access Geocaching.com.For instance, when I try to actualize the currently selected cache, I get an error: "request failed: 401 unauthorized". And I can no more ask for a new API key in the part of the settings menu, where I used to ask for one in such situations.I am a former It teacher and programmer, but I don't want to participate in this cool project as it would require yet another language or environment to be learned. It would be the 18th language in over 44 years of professional life, so I hope you forgive my lazyness in this matterKind regards John Am 25.02.2024 00:52 schrieb Franz Frese ***@***.***>: 
The last version is not yet at the playstore. 
Use dropbox 
http://www.dropbox.com/sh/5jl7hilhlgmzoze/AAAQBsP8YBYjLh3YEifCc91ia?dl=0 
Or better gdrive 
https://drive.google.com/drive/folders/1hKRKMQJ7t-lCNKqK3UbCiQ75oulYRYeD 
Where you can find the latest dev version.

**Comment by tower27** [comment]
Sorry, but what you described here doesn’t work form y installation.

I deinstalled Cachebox and reinstalled it from the app store, this didn’t help.

I installed version 3776 by hand and there is still the same problem

 

 

Thomas

 

Von: Franz Frese ***@***.***> 
Gesendet: Sonntag, 25. Februar 2024 14:32
An: Ging-Buh/cachebox ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [Ging-Buh/cachebox] No more possible to fetch API key (Issue #5)

 

The version is correct. I succeed by
first enter settings, for login, create api key and follow the instructions.

**Comment by arbor95** [comment]
Does your Cachebox show the Version 3776?
Did you click on "Get API Key" from "Login Settings"?

**Comment by Kaujo** [comment]
Yes it shows 3776.! In the login settings, there's no such option anymore. Only 2 items can be found there:1 Password for gcvote2 Fehlende API key holen ohne vorher eine Meldung zu zeigen (fetch API key without showing a message) I know, the "get API Key" once was there! Kind regards JohnAm 25.02.2024 19:18 schrieb Franz Frese ***@***.***>:
Does your Cachebox show the Version 3776?
Did you click on "Get API Key" from "Login Settings"?

**Comment by tower27** [comment]
My version is 3775

 

I always get the message „API Key ist ungültig“ and when I try to create a new api key, nothing happens

 

Von: Franz Frese ***@***.***> 
Gesendet: Sonntag, 25. Februar 2024 19:18
An: Ging-Buh/cachebox ***@***.***>
Cc: tower27 ***@***.***>; Comment ***@***.***>
Betreff: Re: [Ging-Buh/cachebox] No more possible to fetch API key (Issue #5)

 

Does your Cachebox show the Version 3776?
Did you click on "Get API Key" from "Login Settings"?

**Comment by arbor95** [comment]
@tower27 Version 3775 is obsolete and will not work. You must use Version 3776.
@Kaujo  That is not normal but must depend on the data. First I would try with a new Config.db3 (rename or remove the old one). Tell me the directory where your CacheBox is installed. Have a look at the forum https://www.geoclub.de/forum/f/cachebox.114/ . Try with a translation.

**Comment by tower27** [comment]
I couldn’t update to 3776. Nothing happened on my device.

So I deinstalled cachebox and installed Version 3776 manually. But when I try to start this version, the app crashes immediately after starting.

So for the moment I’m not able to use cachebox.

 

Von: Franz Frese ***@***.***> 
Gesendet: Montag, 26. Februar 2024 09:19
An: Ging-Buh/cachebox ***@***.***>
Cc: tower27 ***@***.***>; Mention ***@***.***>
Betreff: Re: [Ging-Buh/cachebox] No more possible to fetch API key (Issue #5)

 

@tower27 <https://github.com/tower27>  Version 3775 is obsolete and will not work. You must use Version 3776.
@Kaujo <https://github.com/Kaujo>  That is not normal but must depend on the data. First I would try with a new Config.db3 (rename or remove the old one). Tell me the directory where your CacheBox is installed. Have a look at the forum https://www.geoclub.de/forum/f/cachebox.114/ . Try with a translation.

**Comment by arbor95** [comment]
I did an update in playstore and just a version 3780 to test on Google drive and Dropbox: download.

@Kaujo: Getting/Updating the key can be done in the settings. If you miss an option in the settings, use the ...-button at the bottom. But getting a new API Key is just clicking onto the line (button) with the description "Get API key" in "Login settings".

and use https://www.geoclub.de/forum/f/cachebox.114/
